static-site/components/logos/index.tsx

```diff
@@ -12,11 +12,17 @@ import bzLogo from "../../static/logos/bz_logo.png";
 import styles from "./styles.module.css";
 
 type Logo = {
```

Comment:
Not a fault of this PR, but I noticed that the `Logo` name is overloaded as both a type and function in this file. It seems to be working fine, but JSDoc can be incorrect (example below shows the function JSDoc on hover of the type):

![image](https://github.com/user-attachments/assets/d3030703-8872-4144-9024-8e2e1008a9f8)

Suggestion: rename to avoid overload. I did something similar with `PhyloTree`/`PhyloTreeType` in Auspice.